scripts/sig_to_csv.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+"""
+Given a sourmash signature, extract the minhash integers and abund info
+to a csv file with the basename of the sig file name as the name of the 
+abund column.
+python sig_to_csv.py 31 signature csv
+"""
+
+from sourmash import signature
+import pandas as pd
+import os
+import sys
+import argparse
+
+def main():
+    p = argparse.ArgumentParser()
```

Comment:
Might wanna check [click](https://click.palletsprojects.com/en/8.1.x/) for creating command-line runnable scripts! It's slightly neater than `argparse` IMO. Or you can try to use the `help` option for `add_argument` in argparse to clean up the comments.

Reply:
for @taylorreiter - we use click in many places - e.g. genome-grist [link](https://github.com/dib-lab/genome-grist/blob/latest/genome_grist/__main__.py). I prefer argparse for complex situations (b/c it's been around longer, has more stackoverflow answers for oddball things) but click is way friendlier!